### ISSUE TITLE
Token Labeling

### ISSUE THREAD

**Issue by monney** [sail-sg/volo#8]
Hi, thank you for the paper.

Do you have any numbers for how the networks perform without token labeling, only using MixToken and other augmentations?

**Comment by yuanli2333** [comment]
Hi, token labeling can improve  volo_d1 by 0.4% (from 83.8% to 84.2%). For larger volo models, token labeling can achieve improvement from 0.0 to 0.4%.

**Comment by monney** [comment]
Thank you, I think these numbers are probably worth adding to the paper as well for context.

**Comment by KleinXin** [comment]
> Hi, token labeling can improve volo_d1 by 0.4% (from 83.8% to 84.2%). For larger volo models, token labeling can achieve improvement from 0.0 to 0.4%.

Could you please also put a copy of token labeling data and pretrained models on Baidu Netdisk？

**Comment by RaySue** [comment]
Hi, thank you for the paper. I got a question: How to get the dense score map by advanced model?

**Comment by Ree1s** [comment]
Hi, if I want to use VOLO to finetune on my own dataset, how can I create my own token labeling?

**Comment by zihangJiang** [comment]
Hi @Ree1s , You can refer to https://github.com/zihangJiang/TokenLabeling/issues/7#issuecomment-863130757 for generating token label data for your own dataset.

**Comment by yuanli2333** [comment]
> Hi,thank you for sharing this work. If I want to use VOLO to finetune on my own dataset, how can I create my own token labeling?

If you only want to 'finetune' VOLO on your own dataset, one option is to directly use the 'class token'  or mean the feature tokens without using token labeling, thus you don't need to generate your own token labels.

**Comment by Ree1s** [comment]
> Hi @Ree1s , You can refer to [zihangJiang/TokenLabeling#7 (comment)](https://github.com/zihangJiang/TokenLabeling/issues/7#issuecomment-863130757) for generating token label data for your own dataset.

Thanks, and I will try on my dataset through this way mentioned.

**Comment by Ree1s** [comment]
> > Hi,thank you for sharing this work. If I want to use VOLO to finetune on my own dataset, how can I create my own token labeling?
> 
> If you only want to 'finetune' VOLO on your own dataset, one option is to directly use the 'class token' or mean the feature tokens without using token labeling, thus you don't need to generate your own token labels.

I'll have a try. Thanks a lot!

**Comment by chengengliu** [comment]
> > Hi,thank you for sharing this work. If I want to use VOLO to finetune on my own dataset, how can I create my own token labeling?
> 
> If you only want to 'finetune' VOLO on your own dataset, one option is to directly use the 'class token' or mean the feature tokens without using token labeling, thus you don't need to generate your own token labels.

Hi, when you say class token, do you mean configure `return_mean` to `True` the feeding parameters of VOLO?
